stl/inc/type_traits

```diff
@@ -1597,6 +1597,8 @@ _CONSTEXPR17 auto invoke(_Callable&& _Obj, _Ty1&& _Arg1, _Types2&&... _Args2) no
 #pragma clang diagnostic ignored "-Wdeprecated-volatile"
 #endif // __clang__
 
+// Note that this is equivalent to xstddef's _Fake_copy_init (and _Fake_decay_copy).
+// TRANSITION, replace _Implicitly_convert_to with _Fake_copy_init.
 template <class _To>
 void _Implicitly_convert_to(_To) noexcept; // not defined

```

Comment:
I disagree strongly here. I believe we should be switching to `_Fake_copy_init` entirely. Having two things which are basically the same seems unnecessary, especially since `_Fake_copy_init` is a good descriptor for both decay-copy and "implicitly convert to", and additionally it uses standard wording. I would much prefer we follow what my comment said, and replace `_Implicitly_convert_to` with `_Fake_copy_init` - otherwise, there's no point in adding `_Fake_copy_init` at all.

Reply:
Reverted, thanks.